.gitignore

```diff
@@ -4,3 +4,5 @@ build/
 /metrictank
 /cmd/mt-split-metrics-by-ttl/mt-split-metrics-by-ttl
 /cmd/mt-store-cat/mt-store-cat
+/cmd/whisperImporter/whisperReader/main
+/cmd/whisperImporter/chunkWriter/main
```

Comment:
The reason they are in a common subdirectory of `cmd` is because they have some shared parts. At the moment the file `cmd/whisperImporter/metric.go` does not have much content, but i was planning to also move the compression/decompression in there. 
How about this structure:
```
/cmd/mt-whisper-importer/metric.go
/cmd/mt-whisper-importer/reader/main.go
/cm/mt-whisper-importer/writer/main.go
```